_includes/layout/navigation.html

```diff
@@ -7,7 +7,7 @@
                         <div class="nav-section">
                                 <div class="spectrum-Menu-sectionHeading">Cloud basics</div>
                                 <ul class="spectrum-Menu">
-                                        <li class="spectrum-Menu-item" role="menuitem"><a href="/cloud/bk-cloud.html"
+                                        <li class="spectrum-Menu-item" role="menuitem"><a href="https://stackoverflow.com/questions/2797091/css-and-and-or"
```

Comment:
This seems odd. Why are we linking to stackoverflow?

Reply:
No, I have added this for testing, because there were no external links there.

Reply:
Reverted, sorry for confusion

Reply:
No problem! It make sense now. Thanks for reverting.